### PR TITLE
Add navbar to all Pages, nav minor bugfixes

### PR DESCRIPTION
**Pull request by daltonfury42**
I was trying adding navbar to all pages.

What triggered this was that we want the login button and all pages. It's easier to just make the navbar common for all pages.

Also some trivial bugfixes. Clicking on logo and "how it works" was not working, fixed those.
Removed some z-indexes along the way and used flexbox.

### CODE REVIEW

File: simplq/src/components/common/Nav/Burger.jsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import React, { useState } from 'react';
 import styles from './nav.module.scss';
 import LeftNav from './LeftNav';
 
-const Burger = ({ page }) => {
```
Comment (daltonfury42) [inline simplq/src/components/common/Nav/Burger.jsx:7]: Made navbar common for all pages, no spl handling per page

File: simplq/src/components/common/Nav/LeftNav.jsx [hunk 2]
```diff
@@ -3,34 +3,34 @@
...
     toggleClose();
     // get the target div by ID
     const element = document.getElementById('target_how_it_works');
-    smoothScrollTo(element);
+    if (element) {
+      // element is on the current page, just have to scroll to it
+      smoothScrollTo(element);
+    } else {
+      window.location.href = '/#target_how_it_works';
+    }
```
Comment (maaverik) [inline simplq/src/components/common/Nav/LeftNav.jsx:19]: Clicking How It Works on another page redirects to the homepage, but doesn't scroll to the exact 'how it works' section, but instead, to the Why SimplQ section.

File: simplq/src/components/common/HeaderSection/headerSection.module.scss [hunk 3]
```diff
@@ -2,23 +2,13 @@
...
   .queue {
-    margin: auto;
-    position: absolute;
-    top: 1rem;
-    bottom: 0;
-    left: 0;
-    right: 0;
     align-items: center;
-    z-index: 1;
     .header-title {
```
Comment (maaverik) [inline simplq/src/components/common/HeaderSection/headerSection.module.scss:12]: How did you manage without z-index? Did the align-items: flex-end fix the problem we were having before?

File: simplq/src/components/common/Nav/LeftNav.jsx (hunk 2 above)
Comment (daltonfury42) [reply simplq/src/components/common/Nav/LeftNav.jsx:19]: Oh is it, I tried here on chrome and it's working. Can you help me reproduce the issue?

File: simplq/src/components/common/HeaderSection/headerSection.module.scss (hunk 3 above)
Comment (daltonfury42) [reply simplq/src/components/common/HeaderSection/headerSection.module.scss:12]: I don't know, I just removed the z-index and everything was working fine. Can you see if you can spot any issues?

I don't exactly remember the reason why we put the z-indices. Maybe the underlying issue got resolved due to some other change.

File: simplq/src/components/common/HeaderSection/index.jsx [hunk 4]
```diff
@@ -1,13 +1,9 @@
 import React from 'react';
 import styles from './headerSection.module.scss';
-import Logo from '../ClickableLogo';
 import Header from '../Header';
 
 export default (props) => (
   <div className={styles['header-bar']}>
-    <div className={styles['simplq-logo']}>
-      <Logo />
-    </div>
```
Comment (daltonfury42) [inline simplq/src/components/common/HeaderSection/index.jsx:10]: We don't need the logo as it's already there on the navbar.

File: simplq/src/components/common/Nav/LeftNav.jsx (hunk 2 above)
Comment (maaverik) [reply simplq/src/components/common/Nav/LeftNav.jsx:19]: https://user-images.githubusercontent.com/12583528/103136152-ecd78580-46e3-11eb-9b69-a84f8ab1a612.mp4


This is how it usually goes for me on Ubuntu with firefox. But I noticed it worked a few times intermittently.

File: simplq/src/components/common/HeaderSection/headerSection.module.scss (hunk 3 above)
Comment (maaverik) [reply simplq/src/components/common/HeaderSection/headerSection.module.scss:12]: Oh okay

File: simplq/src/components/common/Nav/LeftNav.jsx (hunk 2 above)
Comment (daltonfury42) [reply simplq/src/components/common/Nav/LeftNav.jsx:19]: I was able to reproduce this. It seems like a bug with firefox, I've resolved it.

https://github.com/SimplQ/simplQ-frontend/pull/441/commits/2e09215613803c16d16c831edbe4ca524166301b

Comment (maaverik) [reply simplq/src/components/common/Nav/LeftNav.jsx:19]: Cool, looks like it works now.

**Review by maaverik — APPROVED** [review]

**Pull request merged**